docs/7___appendix.adoc

```diff
@@ -216,10 +216,10 @@ Step events are, for example, used to change the mapping of the continuous state
 
 |[[super-dense-time,super-dense time]]_super-dense time_
 |A precise definition of time taking into account iterations at an event.
-For an _FMU_, the <<independent>> variable time latexmath:[\mathbf{t} \in \mathbb{T}] is a tuple latexmath:[\mathbf{t} = (\mathbf{t}_R, \mathbf{t}_I)] where latexmath:[\mathbf{t}_R \in  \mathbb{R}, \mathbf{t}_I \in \mathbb{N} = \{0,1,2,\ldots\}].
-The real part latexmath:[\mathbf{t}_R] of this tuple is the <<independent>> variable of the FMU for describing the continuous-time behavior of the model between events.
-During continuous-time integration latexmath:[\mathbf{t}_I = 0].
-The integer part latexmath:[\mathbf{t}_I] of this tuple is a counter to enumerate (and therefore distinguish) the events at the same continuous-time instant latexmath:[\mathbf{t}_R].
+For an _FMU_, the <<independent>> variable time latexmath:[t \in \mathbb{T}] is a tuple latexmath:[t = (t_R, t_I)] where latexmath:[t_R \in  \mathbb{R}, t_I \in \mathbb{N} = \{0,1,2,\ldots\}].
```

Comment:
t_t does not belong to the natrual number subscipt I does->
change 
t_I \in \mathbb{N}
to I \in \mathbb{N}

Reply:
Yes your are correct I didn't read the rest of the text thoroughly enough.

If we look at table 3
Conventions and notation used for variable subscripts.

We use a different definition for subscript d
t=(ti,n) .
n∈N={0,1,2,…} .

This way of writing is more clear (for me) that t_I is an integer and not a mistake. but i dont require that we change it like that.
However Ithink it could be good if we update eterh here or table 3